### PR TITLE
update price-feed documentation

### PR DESCRIPTION
**Pull request by julio-jimenez**

### CODE REVIEW

File: docs/docs/developer-guides/oracle-set-up-guide.md [hunk 1]
```diff
@@ -31,119 +31,58 @@ Below is a high level diagram that shows the interactions between the components
...
+[Setup a node by following the collator guide](https://docs.composable.finance/developer-guides/collator-guide)
 
 
-### Setting up a node in development mode
+## Setting up the price feed 
 
-For development mode, polkalaunch scripts should be used. 
+### Requirements
+- [Collator node](https://docs.composable.finance/developer-guides/collator-guide/)
+- [Docker](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository) or [Rust](https://www.rust-lang.org/tools/install) if building your own binary
```
Comment (blasrodri) [inline docs/docs/developer-guides/oracle-set-up-guide.md:41]: Perhaps mention that this guide doesn't cover building it from source

**Review by blasrodri — APPROVED** [review]

**Comment by josietyleung** [comment]
hey @dzmitry-lahoda can we merge this thanks

**Comment by dzmitry-lahoda** [comment]
@kkast @bostercf is it still relevant?

**Comment by dzmitry-lahoda** [comment]
> hey @dzmitry-lahoda can we merge this thanks
@josietyleung everything merged as soon as it fixes issue with PR. in this PR docs build fail, may be spelling, may be unproper  format. or old jobs (need rebase).

i do not merge PRs, queue merges these after PR quality is fixes

**Comment by dzmitry-lahoda** [comment]
closing, please reopen if still relevant

**Pull request closed without merging**